5-session-cleaner/main.go

```diff
@@ -20,39 +20,50 @@ package main
 import (
 	"errors"
 	"log"
+	"sync"
+	"time"
 )
 
 // SessionManager keeps track of all sessions from creation, updating
 // to destroying.
 type SessionManager struct {
 	sessions map[string]Session
+	RwMut    sync.RWMutex
```

Comment:
Should be unexported (lower case)

```suggestion
	rwmut    sync.RWMutex
```